gcs/pom.xml

```diff
@@ -208,10 +208,13 @@
               </filters>
               <artifactSet>
                 <includes>
+                  <include>com.fasterxml.jackson.core:jackson-core</include>
```

Comment:
Why do we need Faster XML? We should use Gson instead.

Reply:
Yes, but we use google-api JSON client with Gson, not Jackson. Why do we use Jackson instead of Gson for metrics export?